azure-pipelines.yaml

```diff
@@ -23,34 +23,35 @@ parameters:
     default: 'refs/heads/release'
```

Comment:
Why would the user enter a the branch? would this not come from the system variable $(Build.SourceBranch)